src/pages/index.liquid

```diff
@@ -1,6 +1,6 @@
 ---
 layout: layout.liquid
-title: Verdance Homepage
```

Comment:
nit: can we be consistent with having `Verdance | {pageTitle}` vs. ` {pageTitle} | Verdance`?

Reply:
I vote: `Verdance | {pageTitle}`

Reply:
Sure, I don't have any strong opinions on the  matter and any of the above should be better than just including the URL in the tab